src/core/server/saved_objects/migrationsv2/migrations_state_action_machine.ts

```diff
@@ -85,11 +87,12 @@ export async function migrationStateActionMachine({
   model: Model<State>;
 }) {
   const executionLog: ExecutionLog = [];
-  const starteTime = performance.now();
```

Comment:
@rudolf I'm not sure we need such high precision. I rolled back to the standard Date instead. Let me know if I should revert the change.

Reply:
yeah `Date.now()` is good enough.